apps/core/lib/core/data/model/line_item.ex

```diff
@@ -33,7 +33,7 @@ defmodule Core.Snitch.Data.Model.LineItem do
 
   When both `variant_id` and `quantity` are valid, update is made.
   ```
-  iex> variant = Core.Repo.one(Core.Snitch.Variant)
+  iex> variant = Core.Repo.one(Snitch.Core.Data.Schema.Variant)
```

Comment:
I think you missed renaming the repo namespace.

Reply:
I did that intentionally, is it required to change the namespace of mix, repo, application and config files?